features/lexicon/rollup.config.js

```diff
@@ -67,12 +67,19 @@ export default {
         sucrase({ transforms: [] }),
         copy({
             targets: [
+                {
+                    src: ["node_modules/@polypoly-eu/podjs/dist/pod.js"],
+                    dest: "dist",
+                },
                 {
                     src: [
-                        "node_modules/@polypoly-eu/podjs/dist/pod.js",
                         "node_modules/@polypoly-eu/poly-look/dist/css/poly-look.css",
                     ],
-                    dest: "public",
+                    dest: "dist/css",
+                },
+                {
+                    src: ["src/static/fonts/*"],
+                    dest: "dist/fonts",
```

Comment:
Yes that would be good as well :D. 

As to why the symlink doesn't work when archiving I have a feeling that it is because of the step in which the bundler imports the features as dependencies in its package.json. That means that importing the symbolic-links to the node_modules folder makes the target directory of the link one level deeper so instead of getting to assets/fonts via ../../../../assets/fonts, it needs one more level of backing out. 
Then again, I feel like that should have been a problem in Linux and Mac as well, unless the copying functions take care of that discrepancy under the hood better than windows. 

As for the other features, they have the full folder of the Fonts being copied into dist rather than a symlink so the zipping doesn't have this problem :)

Reply:
Again, the problem is in part because this is using a non-standard way of using the assets. It's not a problem elsewhere because instead of having the symbolic link in a source directory, there's the result of rollup in a dist directory.